### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by theawesomecharles**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,3 +9,4 @@ TrollRAT is a Remote Administration Tool (RAT) which follows a different approac
 I started development, because some people asked me to modify the clean version of MEMZ to be hidden in order to troll tech support scammers. So I thought about a remote controllable version of it. I then thought I should make a whole new project just for trolling which makes it easier to add new payloads without changing MEMZ base concept.
 
 If you want to know what is planned and what already works, look at the [Roadmap](https://github.com/Leurak/TrollRAT/blob/master/Roadmap.md).
+(but you will have to visual studio~)
```
Comment (Nikitaw99) [inline README.md]: Is this really needed? Even I don't understand what this new line means.

Comment (Nikitaw99) [inline README.md]: The line could be changed to "You will need to compile using Visual Studio".

Comment (slashiee) [inline README.md]: I agree, this really needs no purpose, unless the person wants to add a whole segment on how to compile for Visual Studio.

Comment (Energyspooks) [inline README.md]: I agree, no purpose at all.

**Pull request closed without merging**